src/foam/u2/detail/TabbedDetailView.js

```diff
@@ -55,7 +55,7 @@ foam.CLASS({
       this.SUPER();
       this
         .addClass(this.myClass())
-        .add(this.slot(function(sections) {
+        .add(this.slot(function(sections, data) {
```

Comment:
This causes a whole screen flash there is probably some point in this slot where data needs to be slotted and it is not, I would assume L:84

Reply:
If this is only one property then the view property of that property isnt set properly